### ISSUE TITLE
Без тыканья в базу

### ISSUE THREAD

**Issue by reinethernal** [EvilPug/birthday_dog#1]
А как-то без запросов в БД а через бота реализовать добавление дней рождения реально?

**Comment by EvilPug** [comment]
> А как-то без запросов в БД а через бота реализовать добавление дней рождения реально?

Да, планировалось реализовать добавление через бота в телеге, но проект заглох :(
